src/app.service.ts

```diff
@@ -111,7 +112,28 @@ export class AppService {
       .addBearerAuth()
       .build();
 
-    const document = SwaggerModule.createDocument(app, config, {});
+    const document = SwaggerModule.createDocument(app, config, {
+      operationIdFactory: (controllerKey: string, methodKey: string) => {
+        const replacedControllerName = controllerKey.replace(/Controller$/, '');
+
+        const singularControllerName = inflection.singularize(
+          replacedControllerName,
+        );
+
+        const singularMethodName = inflection.singularize(methodKey);
+
+        const methodName = singularMethodName.replace(
+          singularControllerName,
+          '',
+        );
```

Comment:
```suggestion
        const controllerRegExp = new RegExp(singularControllerName, 'i');
        const methodName = singularMethodName.replace(controllerRegExp, '');
```
현재 구조는 controller name이 methodName의 대소문자가 어긋났을 때 대응이 안됩니다.
ex) MajorsController.majorFindAll
생각해보니 MajorsController.findMajors 같은 케이스가 major_find 로 치환될 수 있으니 차라리 methodName은 controller 네임을 지우지 않고 단수형으로 바꾸지도 않는게 나을거같네요